CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+# 0.0.106 - 2022-03-30
+
+## API Updates
+ * Minimum supported rust version (MSRV) is now 1.41.1 (#1310).
+ * `PhantomRouteHints`, `FixedPenaltyScorer`, and `ScoringParameters` now
+   implement `Clone` (#1346).
+ * `DefaultRouter` and `find_route` take an additional random seed to improve
+   privacy by adding a random CLTV expiry offset to each path's final hop. This
+   helps obscure the intended recipient from adversarial intermediate hops
+   (#1286). The seed is  also used to randomize candidate paths during route
+   selection (#1359).
+ * `PeerManager::timer_tick_occurred` will disconnect an unresponsive peer after
+   four ticks, corresponding to 40 seconds with the new recommendation to call
+   the method every 10 seconds (#1348).
+ * The `lightning-block-sync` crate's `init::synchronize_listeners` method
+   interface has been relaxed to support multithreaded environments (#1349).
+ * `ChannelManager::create_inbound_payment_for_hash`'s documentation has been
+   corrected to remove the one-year restriction on `invoice_expiry_delta_secs`,
+   which is only applicable to the deprecated `create_inbound_payment_legacy`
+   and `create_inbound_payment_for_hash_legacy` methods (#1341).
+ * `ChannelManager` supports sending and receiving short channel id aliases in
+   the `funding_locked` message. These are used when forwarding payments and
+   constructing invoice route hints for improved privacy (#1311).
+ * `ChannelDetails` now has a `inbound_scid_alias` field and
+   `get_inbound_payment_scid` method (#1311).
+ * `ChannelManager` no longer fails a channel when receiving redundant
+   `funding_locked` messages since they may be used to update the channel's
```

Comment:
Arguably this is a bugfix but I'm not even sure its worth mentioning. A peer shouldnt send such messages anyway unless we signal a few feature, which we do now but we didn't previously. I don't see how this impacts users.